### PR TITLE
add local-path-storage to the main kurl.sh page

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

**Comment by JustinN-Replicated** [comment]
@paigecalvert a bit curious to see what we could implement from the readme that rancher providers with the component itself here: https://github.com/rancher/local-path-provisioner/
Looking into this now but wanted to share - maybe better if we paired to review together?

**Comment by paigecalvert** [comment]
@JustinN-Replicated I am down to pair!

File: src/markdown-pages/add-ons/local-path-provisioner.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+linktitle: "Local Path Provisioner Add-On"
+weight: 46
+title: "Local Path Provisioner Add-On"
+addOn: "localPathProvisioner"
+---
+
+[Local Path Provisioner](https://github.com/rancher/local-path-provisioner/) is a minimal PVC provisioner that exposes node-local storage.
+
+Local Path Provisioner uses the `/opt/local-path-provisioner` directory for storage on all nodes.
+This directory should have enough space to hold a complete copy of every PersistentVolumeClaim that will be on a given node.
```
Comment (jala-dx) [inline src/markdown-pages/add-ons/local-path-provisioner.md:13]: This begs the question, how much is enough ? is it possible if there is something we can recommend or can we point to Ranchers' doc?

**Review by jala-dx — DISMISSED** [review]

File: src/markdown-pages/add-ons/local-path-provisioner.md (hunk 1 above)
Comment (JustinN-Replicated) [reply src/markdown-pages/add-ons/local-path-provisioner.md:13]: How can we help the user determine how much is enough? We should think about what our typical vendor may need or help guide them on how to figure it out. I would suspect that backups highly impact this (but do not know for sure - snaps that I am used to usually load a delta to the side so for example if it is 50GB and you snap, then it is 50GB + any additional writes to the new snapshot)

Comment (jala-dx) [reply src/markdown-pages/add-ons/local-path-provisioner.md:13]: @laverya as trivial as it sounds, whatever you are saying above has to go to docs i feel!

Comment (laverya) [reply src/markdown-pages/add-ons/local-path-provisioner.md:13]: updated

**Comment by JustinN-Replicated** [comment]
@laverya two quick questions:
1) I noticed that it is k8s 1.12+ in ranchers readme; I don't think we need to list this since we don't go back that far. Just wanted to make sure we all thought the same on this one (IE not worth noting)
2) I noticed the variables in the below; do we need to expose this to the user? May be easier to discuss
image

**Comment by JustinN-Replicated** [comment]
@paigecalvert I think we are good to go here but would like you to approve as well. I think we should be good to go with this but wanted to make sure you get eyes on it as well.

**Review by paigecalvert — DISMISSED** [review]
LGTM. Just added some edits for style and clarity.

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**